urdf/scripts/mesh_distortion/README.md

```diff
@@ -62,15 +70,17 @@ object_prefix = 'coke_can'
 ```
 
 Set optional arguments. If not specified, the default will be used.
+Make sure `method` is specified as a list in brackets, even if it contains only
+one element.
 ```
-fouling_rating = 20
+distort_extent = 0.2
 method = ['subdiv_mod', 'vert_rand', 'edge_subdiv']
 ```
 
 Put the args into the input array:
 ```
 import sys
-sys.argv = ['distort.py', file_path, object_prefix, fouling_rating, method]
+sys.argv = ['distort.py', file_path, object_prefix, distort_extent, method]
 ```
 
 Run the script, replacing its path with the one on your machine:
```

Comment:
Path description in the distorted mesh's `mtl` file uses `absolute` path. The original `coke.mtl` wrote `map_Kd ../materials/coke.png
`. The distorted file produced by the distort.py script writes `map_Kd C:\\Users\\woens\\Downloads\\MeshDistortion\\Coke\\materials\\coke.png`. I think it would be better to keep to the relative.

Reply:
Good catch. I added a `path_mode='RELATIVE'` for the OBJ export, which exposed that in the API https://docs.blender.org/api/current/bpy.ops.export_scene.html#bpy.ops.export_scene.obj 5f7030b

Side note: The COLLADA export (`bpy.ops.wm.collada_export`) though doesn't have that in the API. We aren't using that right now, as it doesn't export textures correctly, so it shouldn't affect the instructions. I added a TODO item above that line.